app/apollo/test/channel.fga.spec.js

```diff
@@ -0,0 +1,666 @@
+    tags: ['tag_02'],
+    contentType: 'local'
+  });
+};
+
+const createVersions = async () => {
+  await models.DeployableVersion.create({
+    _id: 'fake_ver_id_1',
+    org_id: org01._id,
+    uuid: 'testVersion1',
+    name: 'test-version1',
+    channel_id: 'testConfiguration1',
+    channel_name: 'test-configuration1',
+  });
+  await models.DeployableVersion.create({
+    _id: 'fake_ver_id_2',
+    org_id: org01._id,
+    uuid: 'testVersion2',
+    name: 'test-version2',
+    channel_id: 'testConfiguration2',
+    channel_name: 'test-configuration2',
+  });
+};
+
+const createGroups = async () => {
+  await models.Group.create({
+    _id: UUID(),
+    org_id: org01._id,
+    uuid: 'testGroup1',
+    name: 'testGroup1',
+    owner: 'undefined'
+  });
+  await models.Group.create({
+    _id: UUID(),
+    org_id: org01._id,
+    uuid: 'testGroup2',
+    name: 'testGroup2',
+    owner: 'undefined'
+  });
+};
+
+describe('channel graphql test suite', () => {
+  before(async () => {
+    process.env.NODE_ENV = 'test';
+    mongoServer = new MongoMemoryServer( { binary: { version: '4.2.17' } } );
+    await mongoServer.start();
+    const mongoUrl = mongoServer.getUri();
+    console.log(`    cluster.js in memory test mongodb url is ${mongoUrl}`);
+
+    myApollo = await apollo({
+      mongo_url: mongoUrl,
+      graphql_port: graphqlPort,
+    });
+
+    await createOrganizations();
+    await createUsers();
+    await createChannels();
+    await createVersions();
+    await createGroups();
+
+    fgaToken01 = await signInUser(models, resourceApi, fgaUser01Data);
+    fgaToken02 = await signInUser(models, resourceApi, fgaUser02Data);
+  }); // before
+
+  after(async () => {
+    await myApollo.stop(myApollo);
+    await mongoServer.stop();
+  }); // after
+
+  // channels fgaUser01
+  it('fgaUser01 has authentication to get ALLOWED channels', async () => {
+    try {
+      const {
+        data: {
+          data: { channels },
+        },
+      } = await channelApi.channels(fgaToken01, {
+        orgId: org01._id,
+      });
+
+      expect(channels).to.have.length(1);
+
+    } catch (error) {
+      if (error.response) {
+        console.error('error encountered:  ', error.response.data);
+      } else {
+        console.error('error encountered:  ', error);
+      }
+      throw error;
+    }
+  });
+
+  // channels fgaUser02
+  it('fgaUser02 has authentication to get ALLOWED channels', async () => {
+    try {
+      const result = await channelApi.channels(fgaToken02, {
+        orgId: org01._id,
+      });
+
+      expect(result.data.data.channels).to.have.length(1);
+
+    } catch (error) {
+      if (error.response) {
+        console.error('error encountered:  ', error.response.data);
+      } else {
+        console.error('error encountered:  ', error);
+      }
+      throw error;
+    }
+  });
+
+  // channel
+  it('fgaUser01 has authentication to get channel by channel uuid', async () => {
```

Comment:
```suggestion
  it('fgaUser01 has authentication to get channel by channel 1 uuid', async () => {
```

Reply:
Same suggestion should be used on all the test cases descriptions, e.g. `fgaUser01 has authentication to get channel by channel name` should be `fgaUser01 has **authorization** to get channel **1** by channel name` (without the `**` -- they're just there for emphasis in this comment)

Reply:
Changed in all edited fga.spec.js files

Reply:
Resolved